app/src/main/kotlin/com/hippo/ehviewer/gallery/ArchivePageLoader.kt

```diff
@@ -104,6 +106,8 @@ class ArchivePageLoader(
         }
     }
 
+    private fun mayBeAd(index: Int) = index in 0..5 || index in (size - 10)..size
```

Comment:
Given that the false positive rate is so high, I'd say we need to remove the head detection and implement "show blocked image" (through a button or long click).

Reply:
index will always be < size, so we can avoid creating the range.

Reply:
Done